### PR TITLE
Feature/adjust info screen to announce new release features for 2.7

### PR DESCRIPTION
**Pull request by SwissMountainFrog**
This PR brings the new version 27. features info screen
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8295

screenshot27

### CODE REVIEW

**Comment by Ein-Tim** [comment]
@SwissMountainFrog 

In the other versions the bullet points were always **bold**, is this also the case here and the screenshot is not accurate? 

Here's the screen from version 2.5:

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -1588,19 +1588,15 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
+/* Version 2.7 */
 
-"NewVersionFeature_26_euCertCheck_title" = "EU-Zertifikatsprüfung";
+"NewVersionFeature_27_sigCheck_title" = "Signaturprüfung";
 
-"NewVersionFeature_26_euCertCheck_description" = "Sie können nun vor einer Reise Zertifikate für ein gewähltes Land auf Gültigkeit prüfen.";
+"NewVersionFeature_27_sigCheck_description" = "Die Signatur Ihrer Zertifikate wird nun beim Hinzufügen in der Corona-Warn-App auf Gültigkeit und Echtheit geprüft. Damit können nur Zertifikate importiert werden, die von einer autorisierten Stelle ausgestellt wurden.";
 
-"NewVersionFeature_26_enhancedStats_title" = "Erweiterung Statistiken";
+"NewVersionFeature_27_technicalValidity_title" = "Prüfung technische Gültigkeit";
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1597]: "technische Gültigkeit" is a completely new term and users will be unsure what this means. Maybe this should be explained here (IHMO best through a link to a FAQ article "Was ist die technische Gültigkeit?")

@ UA

Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1597]: Hi @Ein-Tim 
Regarding the headlines, thanks for the hint. We already have an own ticket for this. 
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8510

Comment (NataliaLemmerth) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1597]: Hi @Ein-Tim Thanks for your feedback. we'll discuss it.

> "technische Gültigkeit" is a completely new term and users will be unsure what this means. Maybe this should be explained here (IHMO best through a link to a FAQ article "Was ist die technische Gültigkeit?")
> 
> @ UA

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1597]: @NataliaLemmerth @SwissMountainFrog 

Regarding the link, I opened: #3242

I used @SwissMountainFrog's implementation from back in version 1.14 (link to screenshots section). 

I'm really no Swift expert, so please take a look at it and bear with me if I did something wrong. 😅

**Review by naveeddotio — APPROVED** [review]

**Review by pascal-brause — COMMENTED** [review]
@Ein-Tim is right, the headlines and their bullet points should be bold, as it is designed in Figma, too.

**Comment by Ein-Tim** [comment]
@pascal-brause 

Thanks for the feedback, @SwissMountainFrog [told](https://github.com/corona-warn-app/cwa-app-ios/pull/3233#discussion_r675432629) me that this is tracked in EXPOSUREAPP-8510.

**Review by marcussc — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

**Pull request merged**